api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistMediaRepository.kt

```diff
@@ -42,7 +42,7 @@ class AnilistMediaRepository @Inject constructor(
                     seasonYear = Optional.presentIfNotNull(seasonYear)
                 )
             )
-            .fetchPolicy(FetchPolicy.CacheAndNetwork)
+            .fetchPolicy(FetchPolicy.CacheFirst)
```

Comment:
How long is cache valid for, will this still refresh?

Reply:
Still looking into it, we can set the expiry time of the in-memory cache using [`expireTimeMillis`](https://www.apollographql.com/docs/kotlin/caching/normalized-cache#in-memory-cache), but for SQLite we'd have to [`watch()` it](https://www.apollographql.com/docs/kotlin/caching/query-watchers) I think.

Reply:
> By default, there is no timeout.
So until we hit the size limit, or flush

Reply:
Resolving this, the line is gone